### ISSUE TITLE
Anbox is not opening in my pc Garuda linux 

### ISSUE THREAD

**Issue by Dagg1213** [anbox/anbox#1970]
1. Please check that no similar bug is already reported. Have a look on the list of open bugs at https://github.com/anbox/anbox/issues
2. Make sure you are running the latest version of Anbox before reporting an issue. Update snap to latest: `snap refresh --devmode --edge anbox`
3. Make sure you have debug logs enabled:
`sudo snap set anbox debug.enable=true`
4. Reproduce the error while debug logs enabled.
5. Run the anbox logs collection utility and attach the tar file.
`sudo /snap/bin/anbox.collect-bug-info `

6. ** Please paste the result of `anbox system-info` below:**
<details>
<summary>anbox system-info output</summary>

```
[please paste printout of `anbox system-info` here]
```
</details>

**Please describe your problem:**

**What were you expecting?:**

**Additional info:**

**Comment by Dagg1213** [comment]
I installed anbox from Garuda app installer but when I click on anbox app it doesn't open please help me

**Comment by Fuseteam** [comment]
what does `anbox session-manager` say?

**Comment by EduApps-CDG** [comment]
@Fuseteam I have a clean install from snap store. Here is the output:
<pre><font color="#00AA00"><b>[eduardo]:</b></font><font color="#0000AA"><u style="text-decoration-style:single"><b>~</b></u></font><font color="#AA5500"><b>$</b></font> anbox session-manager
[ 2022-01-03 09:22:42] [Renderer.cpp:128@initialize] Failed to select EGL configuration
Stack trace (most recent call last) in thread 36149:
#7    Object &quot;[0xffffffffffffffff]&quot;, at 0xffffffffffffffff, in 
#6    Object &quot;/lib/x86_64-linux-gnu/libc.so.6&quot;, at 0x7f65410e7292, in clone
#5    Object &quot;/lib/x86_64-linux-gnu/libpthread.so.0&quot;, at 0x7f6541458608, in 
#4    Object &quot;/snap/anbox/213/usr/bin/anbox&quot;, at 0x558139ac0314, in emugl::Thread::thread_main(void*)
#3    Object &quot;/snap/anbox/213/usr/bin/anbox&quot;, at 0x5581399cc358, in RenderThread::main()
#2    Object &quot;/snap/anbox/213/usr/bin/anbox&quot;, at 0x558139abb060, in renderControl_decoder_context_t::decode(void*, unsigned long, IOStream*)
#1    Object &quot;/snap/anbox/213/usr/bin/anbox&quot;, at 0x558139a3a562, in 
#0    Object &quot;/snap/anbox/213/usr/bin/anbox&quot;, at 0x558139a3d429, in RendererConfigList::getPackInfo(int*, int*) const
Segmentation fault (Address not mapped to object [(nil)])
Falha de segmentação
</pre>

**Comment by Fuseteam** [comment]
@EduApps-CDG yup sounds like this should help you: https://github.com/anbox/anbox/issues/1943#issuecomment-972905105

**Comment by Fuseteam** [comment]
the solution can be found in #1260
